test_data/std/arith_large_test.asm

```diff
@@ -267,7 +267,8 @@ machine Main with degree: 65536 {
         assert_eq t_0_0, t_0_1, t_0_2, t_0_3, t_0_4, t_0_5, t_0_6, t_0_7, 0x70afe85a, 0xc5b0f470, 0x9620095b, 0x687cf441, 0x4d734633, 0x15c38f00, 0x48e7561b, 0xd01115d5;
         assert_eq t_1_0, t_1_1, t_1_2, t_1_3, t_1_4, t_1_5, t_1_6, t_1_7, 0xf4062327, 0x6b051b13, 0xd9a86d52, 0x79238c5d, 0xe17bd815, 0xa8b64537, 0xc815e0d7, 0xa9f34ffd;
 
-        // Auto-generated rest of the test cases:
+        // Auto-generated rest of the test cases,
+        // see: https://gist.github.com/georgwiese/a66425ae95c839b548edffffe6a692f9
```

Comment:
Thankfully, I found an old Python notebook that used to generate these tests. I uploaded them to Gist and linked it here for future reference. The script also contains code to generate the memory tests.